### PR TITLE
Homework #2 is completed

### PR DESCRIPTION
**Pull request by Syzygy9**
Please check Homework#2

### CODE REVIEW

File: src/test/java/hw2/PreAssertionClass.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+
+public class PreAssertionClass {
```
Comment (DmitryKhodakovsky) [inline src/test/java/hw2/PreAssertionClass.java]: Why do you decide choose this strange name for the class?

File: src/test/java/hw2/ex2/AssertElementsTest.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+       */
+        WebElement elementWater = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li[contains(text(),'Water: condition changed to true')]")));
+        assertTrue(elementWater.isDisplayed(), "Water checkbox is not displayed in log row");
+        WebElement elementWind = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li[contains(text(),'Wind: condition changed to true')]")));
+        assertTrue(elementWind.isDisplayed(), "Wind checkbox is not displayed in log row");
+        WebElement elementSelen = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li[contains(text(),'metal: value changed to  Selen')]")));
+        assertTrue(elementSelen.isDisplayed(), "Selen radio button is not displayed in log row");
+        WebElement elementYellow = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li[contains(text(),'Colors: value changed to Yellow')]")));
+        assertTrue(elementYellow.isDisplayed(), "Dropdown selected color is not displayed in log row");
```
Comment (DmitryKhodakovsky) [inline src/test/java/hw2/ex2/AssertElementsTest.java]: Why do you put full log text into locator? Please try to find appropriate locator

File: src/test/resources/suites/hw2/AllTests.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="All the tests suite">
```
Comment (DmitryKhodakovsky) [inline src/test/resources/suites/hw2/AllTests.xml]: This suite does not work

File: src/test/resources/suites/hw2/Exercise1.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```
Comment (DmitryKhodakovsky) [inline src/test/resources/suites/hw2/Exercise1.xml:1]: This suite does not work

File: src/test/resources/suites/hw2/Exercise2.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```
Comment (DmitryKhodakovsky) [inline src/test/resources/suites/hw2/Exercise2.xml:1]: This suite does not work

**Review by DmitryKhodakovsky — CHANGES_REQUESTED** [review]
Please resolve comments ASAP. Tests does not work though the command `mvn clean test -P hw2-all-tests`

**Comment by Syzygy9** [comment]
I have made the corrections. Please check.

File: src/test/java/hw2/ex2/AssertElementsTest.java (hunk 2 above)
Comment (DmitryKhodakovsky) [reply src/test/java/hw2/ex2/AssertElementsTest.java]: Why do you ignore my comments?

**Review by DmitryKhodakovsky — APPROVED** [review]
HW2 is accepted with comment,. could be merged to the master branch

**Pull request merged**